diagnosis/README.md

```diff
@@ -36,4 +46,6 @@ The logs retrieved by `getkuberneteslogs.sh` are the following:
 `--api-model`                  - Persists apimodel.json file in an Azure Stack Storage account. 
                                  Upload apimodel.json file to storage account happens when `--upload-logs` parameter is also provided.
 
+`--include-windows-nodes`      - Collect log for Windows nodes. Only apply to scenarios when Windows nodepools exist in the Kubernetes clusters.
```

Comment:
I would try to infer this instead of asking the user to set the flag

Reply:
addressed